src/content/peerconnection/negotiate-timing/test/test.js

```diff
@@ -0,0 +1,45 @@
+/*
```

Comment:
is this test needed? They're in bad shape anyway

Reply:
Nuking the test directory. I copied it without thinking.